isis/src/mro/apps/marcical/marcical.cpp

```diff
@@ -524,6 +512,11 @@ namespace Isis {
       icubeMgr++;
       ocubeMgr++;
 
+      // checks if we have reached the end before setting higher band
+      if (ocubeMgr.end()) {
```

Comment:
Prevents non-existent band from being set in the last chunk of code in the loop.